rt/transports/websocket/src/main/java/org/apache/cxf/transport/websocket/undertow/WebSocketUndertowServletRequest.java

```diff
@@ -570,14 +571,31 @@ public <T extends HttpUpgradeHandler> T upgrade(Class<T> arg0) throws IOExceptio
         throw new UnsupportedOperationException();
     }
 
-    @Override
+    
     public String getRealPath(String path) {
         return path;
     }
 
-    @Override
+
     public boolean isRequestedSessionIdFromUrl() {
         LOG.log(Level.FINE, "isRequestedSessionIdFromUrl");
         return false;
     }
+
+    public String getRequestId() {
+        // TODO Auto-generated method stub
```

Comment:
Same, let's get rid of this comments please

Reply:
a new commit addressed it